doc/src/modules/vector/basics.rst

```diff
@@ -0,0 +1,254 @@
+============================
+Basic Implementation details
+============================
+
+Coordinate Systems and Vectors
+==============================
+
+As of now, :mod:`sympy.vector` only deals with the Cartesian (also called 
+rectangular) coordinate systems. A 3D Cartesian coordinate system can
+be initialized in :mod:`sympy.vector` as
+
+  >>> from sympy.vector import CoordSysCartesian
+  >>> N = CoordSysCartesian('N')
```

Comment:
I just thought the names were getting a little too long. And yeah, that was the idea behind calling it the 'rectangular' system.
So should I go with `RectangularCoordinateSystem'?